### PR TITLE
implemented basic logic to assign a random recipient

### PR DESCRIPTION
**Pull request by wolkemann**

### CODE REVIEW

File: pages/api/mails/index.js [hunk 1]
```diff
@@ -16,9 +17,18 @@ export default async function handler(request, response) {
 
       case "POST":
         if (session) {
+          const selectUser = await User.find({ _id: { $ne: session.user.id } });
+          console.log(selectUser.length);
```
Comment (NikRoe) [inline pages/api/mails/index.js:21]: you could remove the console.log

File: pages/api/mails/index.js [hunk 2]
```diff
@@ -16,9 +17,18 @@ export default async function handler(request, response) {
...
+
+          const assignRecipient =
+            selectUser[Math.floor(Math.random() * selectUser.length) + 0];
+
+          console.log(assignRecipient);
```
Comment (NikRoe) [inline pages/api/mails/index.js:26]: as above

File: pages/signin/success.js [hunk 3]
```diff
@@ -30,8 +30,13 @@ export default function SuccessMessage() {
       const updateUser = await response.json();
       console.log(updateUser);
```
Comment (NikRoe) [inline pages/signin/success.js:31]: as above :)

File: schemas/Reply.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { Schema, model } from "mongoose";
+
+const mailSchema = new Schema(
+  {
+    text: { type: String, required: true, minlength: 10 },
+    authorId: { type: Schema.Types.ObjectId, ref: "User" },
+    recipientId: { type: Schema.Types.ObjectId, ref: "User", default: null },
+    reply_to_mail: { type: Schema.Types.ObjectId, ref: "Mail", default: null },
+    reactionEmonji: { type: String },
```
Comment (NikRoe) [inline schemas/Reply.js:9]: should this not be
reactionEmoji: { type: String },
?

**Review by NikRoe — CHANGES_REQUESTED** [review]
looks really good!

there are some console.logs left in your code
and in the mailSchema there might be a slight typo

File: schemas/Reply.js (hunk 4 above)
Comment (wolkemann) [reply schemas/Reply.js:9]: On the prowl, thanks!

File: components/Letter/Letter.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import useSWR from "swr";
+
+export default function Letter({ children, authorId }) {
+  const author = useSWR(`/api/user/${authorId}`);
+
+  return (
+    <FormWrapper>
+      {author.data ? (
+        <>
```
Comment (flohip) [inline components/Letter/Letter.js:10]: do you need the fragments, if you have the FormWrapper?

**Review by flohip — APPROVED** [review]

**Pull request merged**